leedsgeeks.json

```
@@ -77,6 +77,15 @@
                 {"Label": "Twitter", "URL": "https://twitter.com/LeedsCodeDojo", "LinkText": "@LeedsCodeDojo"}
             ]
         },
+        {
+            "Name": "Leeds Data Platform User Group",
+            "Description": "A group for anyone interested in learning more and / or sharing their knowledge of the Microsoft Data Platform. All skill levels are welcome. Topics and technologies will be varied but along the lines of SQL Server, SSIS, SSAS, Power BI, Azure DB, Azure automation, Azure SQL Data Warehouse, SQL Server on Linux and containers",
+            "Links": [
+                {"Label": "Web", "URL": "https://www.meetup.com/leeds_data_platform/"},
+                {"Label": "Twitter", "URL": "https://twitter.com/leedsdata", "LinkText": "@leedsdata"}
```

It looks like there's a `,` missing from the end of this line.

```suggestion
                {"Label": "Twitter", "URL": "https://twitter.com/leedsdata", "LinkText": "@leedsdata"},
```